.travis.yml

```diff
@@ -18,12 +18,6 @@ before_install:
   # Removes Travis-specific PATH customizations that affect Python.
   - export PATH="$(echo $PATH | tr ':' "\n" | sed '/\/opt\/python/d' | tr "\n" ":" | sed "s|::|:|g")"
   - printenv | sort # dump Travis environment for debugging
-  - sudo apt-get update -qq
-  - sudo apt-get install --yes rng-tools
-  - sudo rm -f /dev/random
-  - sudo mknod -m 0666 /dev/random c 1 9
-  - echo HRNGDEVICE=/dev/urandom | sudo tee /etc/default/rng-tools
-  - sudo /etc/init.d/rng-tools restart
```

Comment:
It looks like this was designed to replace /dev/random with /dev/urandom so it does not block because there is not enough entropy. Am I mistaken ?

Reply:
`/dev/urandom` exists on the newer Trusty images provided by Travis. The original commit implementing the /dev/random workaround is 0b18cd15392e0000cda8bbe6dc4d10bffd81a276, nearly 3 years ago. I would prefer to drop it and only re-add it if we notice the tests slowing down.